src/components/Goals.astro

```diff
@@ -1,53 +1,42 @@
 ---
+import { getLangFromUrl, useTranslations } from '../i18n/utils';
 
+const lang = getLangFromUrl(Astro.url);
+const t = useTranslations(lang);
```

Comment:
I would prefer to avoid single letter variables/functions

Reply:
Normally I really do agree but its used throughout the pages and I think its fine to be short as you could argue it actually increases readability?

Reply:
I still think isabel is more correct here. We could call it something like translation(), which I'd argue is more readable than t()

Reply:
> The [astro-i18next](https://github.com/yassinedoghri/astro-i18next) extension or [astro's i18n docs](https://docs.astro.build/en/recipes/i18n/) also use t.

Also this, but its fine.